src/sparql/get-terms.sparql.jinja2

```diff
@@ -0,0 +1,10 @@
+prefix ICD10CM: <http://purl.bioontology.org/ontology/ICD10CM/>
+prefix owl:  <http://www.w3.org/2002/07/owl#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+
+select ?term_id ?term_label where {
```

Comment:
Why is this needed, if you already supply you with the term tables?

Reply:
Ah, good point, thanks Nico. I probably would have realized this when I got further on w/ my script, but now that you bring it up, I realize that this particular query is not necessary at all. Just need to the other query to get children. Don't need this query, as I already have `term_id` and `term_label` for the parent classes excluded.